src/Servers/Kestrel/Transport.Quic/src/Internal/QuicTransportContext.cs

```diff
@@ -3,19 +3,19 @@
 
 using Microsoft.Extensions.Hosting;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Transport.MsQuic.Internal
+namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.Internal
 {
-    internal class MsQuicTransportContext
+    internal class QuicTransportContext
     {
-        public MsQuicTransportContext(IHostApplicationLifetime appLifetime, IMsQuicTrace log, MsQuicTransportOptions options)
+        public QuicTransportContext(IHostApplicationLifetime appLifetime, IQuicTrace log, QuicTransportOptions options)
         {
             AppLifetime = appLifetime;
             Log = log;
             Options = options;
         }
 
         public IHostApplicationLifetime AppLifetime { get; }
```

Comment:
Is the IHostApplicationLifetime still being used anywhere? It seems like a pretty weird requirement for stuff like QuicConnectionFactory.